test_package/test_package/timerange.py

```diff
@@ -247,27 +247,40 @@ def __repr__(self):
         fully_qualified_name = f"{self.__class__.__module__}.{self.__class__.__name__}"
 
         return (
-            "   <{} object at {}>".format(fully_qualified_name, hex(id(self)))
-            + "\n    Start:".ljust(12)
-            + t1
-            + "\n    End:".ljust(12)
-            + t2
-            + "\n    Center:".ljust(12)
-            + center
-            + "\n    Duration:".ljust(12)
-            + str(self.days.value)
-            + " days or"
-            + "\n    ".ljust(12)
-            + str(self.hours.value)
-            + " hours or"
-            + "\n    ".ljust(12)
-            + str(self.minutes.value)
-            + " minutes or"
+            (
+                (
+                    (
+                        (
+                            (
+                                (
+                                    (
+                                        f"   <{fully_qualified_name} object at {hex(id(self))}>"
+                                        + "\n    Start:".ljust(12)
+                                    )
+                                    + t1
+                                )
+                                + "\n    End:".ljust(12)
+                                + t2
+                            )
+                            + "\n    Center:".ljust(12)
+                            + center
+                        )
+                        + "\n    Duration:".ljust(12)
+                        + str(self.days.value)
+                        + " days or"
+                    )
+                    + "\n    ".ljust(12)
+                    + str(self.hours.value)
+                    + " hours or"
+                )
+                + "\n    ".ljust(12)
+                + str(self.minutes.value)
+                + " minutes or"
+            )
             + "\n    ".ljust(12)
             + str(self.seconds.value)
             + " seconds"
-            + "\n"
-        )
+        ) + "\n"
```

Comment:
This is a war crime.

Reply:
This was fun to type. I will see you at The Hague